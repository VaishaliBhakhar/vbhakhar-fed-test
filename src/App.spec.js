import React from 'react';
import App from './App';

describe('App', () => {
	it('renders the App', () => {
		const actual = shallow(<App />);
		expect(actual.get(0)).toMatchSnapshot();
	});
});
