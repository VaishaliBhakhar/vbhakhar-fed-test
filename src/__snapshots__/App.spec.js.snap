// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the App 1`] = `<User />`;
